src/main/java/pacman/Dots.java

```diff
@@ -0,0 +1,14 @@
+class Dots extends Position {
+    
+    private boolean special;
```

Comment:
Also, it'd be great if could be renamed to isSpecial

Reply:
Thanks, the name has been renamed   5b31b5f